source/wp-content/themes/wporg-developer-2023/inc/breadcrumb-trail.php

```diff
@@ -855,6 +857,24 @@ public function do_handbook_search_items() {
 		}
 	}
 
+	/**
+	 * Adds the items to the trail items array for wp-cli commands search results.
+	 *
+	 * @since  0.6.0
+	 * @access public
+	 * @return void
+	 */
+	public function do_cli_commands_search_items() {
```

Comment:
This seems to be very similar to ` do_handbook_search_items`. Looks like the only difference is the home URL. Can we dry that up? Maybe pass in the home URL?